src/components/board/Boards.tsx

```diff
@@ -1,6 +1,21 @@
+import React from "react";
+import Board from "./Board";
+import { Members } from "@/data/BoardMembers";
+
 const Boards = () => {
   return (
-    <div className="flex w-full flex-row justify-center text-3xl">Boards</div>
+    <div className="grid grid-cols-2 gap-x-[15%] gap-y-[6%] p-[10%] pb-[90%] pt-[8%] md:grid-cols-3 md:gap-[10%] md:pb-[50%]">
```

Comment:
I think if you use `w-5/6` then it will remove the need for padding on the left and right sides.